app/models/player.rb

```diff
@@ -7,7 +7,7 @@ class Player < ActiveRecord::Base
 
   after_create do
     # generate score record
-    Score.create(wins: 0, loses: 0, points: 0, goals: 0, wRatio: 0, pullUps: 0, player_id: self.id)
+    Score.create(wins: 0, loses: 0, points: 0, goals: 0, wRatio: 0, pullUps: 0, win_streak: 0, longest_win_streak: 0, player_id: self.id)
```

Comment:
Strictly speaking `player.rb` shouldn't be concerned with how `score.rb` is implemented, so try to keep attributes within their associated models.

The benefit of this is that if someone new actually updates the `score` model (eg. add new attribute to that model), it is much easier for all the changes associated to that model to be in the relevant files (ie. `score.rb`, `score_controller.rb`, etc.), rather than do a global search of every file. 

What you can try to do is:
```ruby
# score.rb - add in a set_defaults method
after_initialize :set_defaults, if: :new_record?

def set_defaults
  [:wins, :loses, :points, :goals, :wRatio, :pullUps, :win_streak, :longest_win_streak].each do |attribute|
    attribute ||= 0 
  end
end

# player.rb
after_create do
  Score.build(player: self).save!
  puts '...'
end
```